cmake/FindCUDA_advanced.cmake

```diff
@@ -13,6 +13,8 @@ if(CUDA_FOUND)
   set(CUDA_NVCC_FLAGS75 "-gencode arch=compute_75,code=sm_75")
   set(CUDA_NVCC_FLAGS8 "-gencode arch=compute_80,code=sm_80")
   set(CUDA_NVCC_FLAGS86 "-gencode arch=compute_86,code=sm_86")
+  set(CUDA_NVCC_FLAGS87 "-gencode arch=compute_87,code=sm_87")
```

Comment:
Should we discontinue some of the older ones here to keep the compile time down, do er really need to go back to 6?

Reply:
retire compute_6x and _7x.